web-client/src/ustc-ui/Tabs/Tabs.tsx

```diff
@@ -45,17 +45,18 @@ const renderTabFactory = ({
 
     const HeadingElement = headingLevel ? `h${headingLevel}` : 'span';
     const tabProps = {
-      'aria-controls': tabContentId,
-      'aria-selected': isActiveTab,
       className: liClass,
-      role: 'tab',
+      role: 'presentation',
     };
 
     const buttonProps = {
+      'aria-controls': tabContentId,
+      'aria-selected': isActiveTab,
       className: childClassName,
       disabled,
       id: buttonId,
       onClick: () => setTab(tabName),
+      role: 'tab',
```

Comment:
In our DOM tree, we were generating something like the following:
```
<ul role="tablist">
  <li aria-controls=... aria-selected=..., role='tab'>
    <button>Blah</button>
  </li>
  ...
</ul>
```

The issue here is that `button` is by nature interactive, and it was being nested within an `li` that was interactive because of its role as a tab child element. I fixed this by moving the `role`, `aria-control`, and `aria-selected` information to the `button` element and signaling that the `li` element is non-semantic (in other words, just used for display) with `role='presentation'`.